### PR TITLE
Create Tier3 of the taxonomy

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Review by escattone — CHANGES_REQUESTED** [review]

File: kitsune/products/migrations/0021_auto_20241120_0218.py [hunk 1]
```diff
@@ -0,0 +1,627 @@
...
+        ],
+        "parent_slug": "manage-subscriptions",
+    },
+    {
+        "title": "Customized reader",
+        "slug": "customized-reader",
+        "description": "Troubleshoot customized reader issues.",
+        "products": [],
+        "parent_slug": "article-view",
+    },
```
[the remark this answers is not in the document]
Comment (akatsoulas) [reply kitsune/products/migrations/0021_auto_20241120_0218.py:124]: I agree with you, I am going to raise this to the taxonomy meeting. We have a total of 9 tier 3 topics that are not associated with any product. My opinion is that we need to drop them. For now I am just keeping the code in sync with the mapping. I will update this either by removing the topics or by adding products

File: kitsune/products/migrations/0021_auto_20241120_0218.py [hunk 2]
```diff
@@ -0,0 +1,627 @@
...
+                "description": entry["description"],
+                "display_order": 50,
+                "visible": True,
+            },
+        )
+
+        try:
+            parent = Topic.objects.get(slug=entry["parent_slug"], is_archived=False)
+        except Topic.DoesNotExist:
+            continue
```
[the remark this answers is not in the document]
Comment (akatsoulas) [reply kitsune/products/migrations/0021_auto_20241120_0218.py:605]: Good call, I moved the query at the top. This way we will skip the tier3 topic creation altogether if there's not a tier2 parent.

**Comment by akatsoulas** [comment]
@escattone PR udpated

File: kitsune/products/migrations/0021_auto_20241120_0218.py [hunk 3]
```diff
@@ -0,0 +1,631 @@
...
+        "description": "Troubleshoot web certificates issues.",
+        "products": ["firefox", "mobile", "ios"],
+        "parent_slug": "error-codes",
+    },
+    {
+        "title": "Blocked application/service/website",
+        "slug": "blocked-application-service-website",
+        "description": "Troubleshoot blocked application/service/website issues.",
+        "products": ["firefox", "mobile", "ios", "firefox-private-network-vpn"],
+        "parent_slug": "site-brekages",
```
Comment (escattone) [inline kitsune/products/migrations/0021_auto_20241120_0218.py:407]: Typo.
```suggestion
        "parent_slug": "site-breakages",
```

File: kitsune/products/migrations/0021_auto_20241120_0218.py [hunk 4]
```diff
@@ -0,0 +1,631 @@
...
+    },
+    {
+        "title": "Security software",
+        "slug": "security-software",
+        "description": "Troubleshoot security software issues.",
+        "products": ["firefox", "mobile", "ios", "firefox-private-network-vpn"],
+        "parent_slug": "site-brekages",
```
Comment (escattone) [inline kitsune/products/migrations/0021_auto_20241120_0218.py:414]:
```suggestion
        "parent_slug": "site-breakages",
```

File: kitsune/products/migrations/0021_auto_20241120_0218.py [hunk 5]
```diff
@@ -0,0 +1,631 @@
...
+    },
+    {
+        "title": "Web compatibility",
+        "slug": "web-compatibility",
+        "description": "Troubleshoot web compatibility issues.",
+        "products": ["firefox", "mobile", "ios", "firefox-private-network-vpn"],
+        "parent_slug": "site-brekages",
```
Comment (escattone) [inline kitsune/products/migrations/0021_auto_20241120_0218.py:421]:
```suggestion
        "parent_slug": "site-breakages",
```

**Review by escattone — APPROVED** [review]
Just some typos to fix, and this is good to go! Thanks for this @akatsoulas !

File: kitsune/products/migrations/0021_auto_20241120_0218.py (hunk 3 above)
Comment (escattone) [reply kitsune/products/migrations/0021_auto_20241120_0218.py]: I added a commit to fix these typos.

File: kitsune/products/migrations/0021_auto_20241120_0218.py (hunk 4 above)
Comment (escattone) [reply kitsune/products/migrations/0021_auto_20241120_0218.py]: I added a commit to fix these typos.

File: kitsune/products/migrations/0021_auto_20241120_0218.py (hunk 5 above)
Comment (escattone) [reply kitsune/products/migrations/0021_auto_20241120_0218.py]: I added a commit to fix these typos.

**Pull request merged**